ui/src/app/intro/components/IntroCard/IntroCard.tsx

```diff
@@ -0,0 +1,58 @@
+import type { HTMLProps, ReactNode } from "react";
+
+import { Card, Icon, Spinner } from "@canonical/react-components";
+
+type Props = {
+  children: ReactNode;
+  complete?: boolean;
+  hasErrors?: boolean;
+  loading?: boolean;
+  title: ReactNode;
+  titleLink?: ReactNode;
+} & Omit<HTMLProps<HTMLDivElement>, "title">;
+
+const IntroCard = ({
+  children,
+  complete,
+  hasErrors,
+  loading,
+  title,
+  titleLink,
+  ...props
+}: Props): JSX.Element => {
+  if (loading) {
+    return <Spinner />;
+  }
```

Comment:
Does this need a `loading` prop? I feel like `IntroSection` should handle that

Reply:
You're right, I was thinking individual cards might need their own loading states, but nothing does currently so I'll remove this.